tests/base/CMakeLists.txt

```diff
@@ -15,4 +15,7 @@ add_executable(${exe}
   # add more test files if appropriate
 )
 target_link_libraries(${exe} PRIVATE Catch2::Catch2WithMain)
-catch_discover_tests(${exe} TEST_PREFIX "${suite}::")
+catch_discover_tests(${exe}
+  TEST_PREFIX "${suite}::"
+  DL_PATHS ${LD_LIBRARY_PATH}
```

Comment:
I would propose to add the DL_PATHS only to tests where we really need it. Currently this is only the test in simulation/Tutorial1.
The advantage I see is that we will see in that way if we link a dependency which have no proper RPATH information. If we add the DL_PATHS information in any case the piece of code is used everywhere and we wouldn't see the problem.

Reply:
done